site/book/03-packages/01-getting-a-package.md

```diff
@@ -0,0 +1,118 @@
+Packaging in kpt based on Git forking. Packages are published by committing them to a Git repository
+and packages are consumed by creating a new fork of the package.
+
+Let's revisit the Wordpress example:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/wordpress@v0.1
+```
+
+A package on a Git repo can be fetched by specifying a branch, tag, or a commit SHA. In this case,
+we are specifying tag `v0.1`.
+
+> Refer to the [command reference][get-doc] for more details.
```

Comment:
is this a quote?  Using quoting markdown.  Not sure what is quoted.

Reply:
This is a hack for generating a Notes/Info box.  We need a proper solution here @etefera. Hugo supported this. e.g. see `pageinfo` directive in 

https://github.com/GoogleContainerTools/kpt/blob/master/site/content/en/guides/consumer/set/_index.md

What's the equivalent way to do it with the new site?

Reply:
We can simulate the `pageinfo` directive if need be, but this is an example of the lock-in we may want to avoid. If the approach here is *too* hacky, it may be preferable to use `<div class="pageinfo">` directly so that the rendered markdown is more portable. We can then style it independently or process it however else we want.